contracts/standard/arbitration/TwoPartyArbitrable.sol

```diff
@@ -55,8 +54,7 @@ contract TwoPartyArbitrable is Arbitrable {
         timeout=_timeout;
         partyA=msg.sender;
         partyB=_partyB;
-        metaEvidence=_metaEvidence;
-        emit MetaEvidence(_metaEvidence);
+        emit MetaEvidence(0, _metaEvidence);
```

Comment:
I couldn't spot any bug everything looks good. Only thing is I couldn't understand this zero in `emit MetaEvidence(0, _metaEvidence)`. Why do we emit an event with fixed zero?

Reply:
We need to use an ID so that contracts that support multiple arbitrable contracts/disputes can reference the correct meta-evidence. In the case of `TwoPartyArbitrable` there should only ever be one meta-evidence event that is emitted on contract creation. So I just hardcoded the ID to be 0 here and also in the create the `LinkMetaEvidence` event so that the dispute always uses the meta-evidence from the contract creation